modules/gli_auth0_profile/js/registration.js

```diff
@@ -37,12 +37,16 @@
         childList: true
       });
 
+      let previousHeight = 96;
       const resizeObserver = new ResizeObserver((entries) => {
         for (const entry of entries) {
-          // Scroll up when the form is updated.
-          if (window.scrollY > 0) {
+          // Scroll up when the form is updated. Only trigger the scroll on
+          // large layout shifts.
+          const newHeight = entry.contentBoxSize[0].blockSize;
```

Comment:
```suggestion
          const newHeight = entry.contentBoxSize[0]?.blockSize ?? 0;
```

This is my thought just incase we don't run into any possible errors

Reply:
its weird the [API doc](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver) examples sometimes check for it, sometimes not. But better safe then sorry